test/fixtures/config/groups.js

```diff
@@ -0,0 +1,11 @@
+const faker = require('faker');
+
+module.exports = {
+  generate: () => {
```

Comment:
this won't fail the linter, but `=>` is giving this function the context of the file, but isn't necessary.  On a POJO, unless context is needed, this can just be `generate() {`